src/containers/AppPage/Settings.js

```diff
@@ -0,0 +1,139 @@
+// Medium pink: #FF8FB3
+// Lighter pink: #F9DAE4
+// Sidebar pink: #FDF6F8
+// Grey: #EEEEEE
+
+// Circle: #FFAEC8
+// Lighter circle: #F9DAE4
+
+// Dark line: #AB4A78
+
+class Settings extends Component {
+    constructor() {
+        super()
+        this.state = {
+            currentBudget: "",
+            budgets: [
+              { month: 8, budget: 5000.00, spendings: 5510.00},
+              { month: 9, budget: 4000.00, spendings: 4567.89},
+              { month: 10, budget: 2500.00, spendings: 2500.00},
+              { month: 11, budget: 1500.00, spendings: 1000.00}
+           ]
+        }
+        this.handleChange = this.handleChange.bind(this)
+      }
+
+   handleChange(event) {
+       const {name, value} = event.target
+       this.setState({[name]: value})
+    }
+
+    handleSubmit(event) {
+       // if (event.target.password.value !== event.target.retype.value){
+            event.preventDefault()
+            alert("Your budget has been set.")
+            this.currentBudget.value ="";
+
+      // }
+       //else if (if form fields are empty){
+           //event.preventDefault()
+           //put alert
+       //}
+    //insert code to send information to database      
+    }
+
+    getDateMonth(monthNumber){
+      var month = new Array;
+      month[0] = "January";
+      month[1] = "February";
+      month[2] = "March";
+      month[3] = "April";
+      month[4] = "May";
+      month[5] = "June";
+      month[6] = "July";
+      month[7] = "August";
+      month[8] = "September";
+      month[9] = "October";
+      month[10] = "November";
+      month[11] = "December";
+      return month[monthNumber];
+    }
+
+    tableHeader(){
+      let header = Object.keys(this.state.budgets[0])
+      return header.map((key, index) => {
+         return <th key={index}>{key.toUpperCase()}</th>
+      })
+    }
+
+    pullPastBudgets() {
+      return this.state.budgets.map((budgets, index) => {
+         const { month, budget, spendings } = budgets
+         if (month < new Date().getMonth()){
+         return (
+            <tr key={month}>
+               <td>{this.getDateMonth(month)}</td>
+               <td>{"$" +  budget}</td>
+               <td>{"$" + spendings}</td>
+            </tr>
+         )
+         }
+      })
+   }
+
+   pullFutureBudgets() {
+    return this.state.budgets.map((budgets, index) => {
+       const { month, budget, spendings } = budgets
+       if (month > new Date().getMonth()){
+        return (
+           <tr key={month}>
+              <td>{this.getDateMonth(month)}</td>
+              <td>{"$" + budget}</td>
+              <td>{"$" + spendings}</td>
+           </tr>
+        )
+        }
+    })
+ }
+
+    render(){
+        return (
+            <AppPage>
+              <form onSubmit={this.handleSubmit}>
+                  <h1 style={{color: "#FF8FB3"}}>SETTINGS</h1>
+                  <h3 style={{color: "#F9DAE4"}}>Current Budget ({this.getDateMonth(new Date().getMonth())})</h3>
```

Comment:
here would probably be month[Date().getMonth()]

Reply:
i never set a variable to be the new Date() value so I've been creating essentially new variables every time to get the most updated date which is why it says new Date() everywhere (I also tried removing new and it didn't work)

would it be better if I create a new Date() variable and then use that throughout the code? the most updated date isn't necessarily needed, I only added it to get the current month

Reply:
the getMonth function gave me the number of the month so I called the method I created above to provide the name of the month at the header section -- is that what you're referring to?